source/kraken/tests/apply_disruption_test.cpp

```diff
@@ -2312,6 +2312,20 @@ BOOST_AUTO_TEST_CASE(test_disruption_on_line_then_stop_point) {
                               .get_disruption(),
                               *b.data->pt_data, *b.data->meta);
 
+    navitia::apply_disruption(b.impact(nt::RTLevel::RealTime, "Y'a plus qu'a attendre ton train")
```

Comment:
nothing keeps us from doing so, but in reality, it doesn't make sense when you apply `SIGNIFICANT_DELAYS` without new datetimes :)

Reply:
you may take look in one of the previous tests how to handle SIGNIFICANT_DELAYS and mix those two realtime level...

Reply:
I've removed the SIGNIFICANT_DELAYS as they were not affecting the test.